src/ElmSharp/ElmSharp/EcoreAnimator.cs

```diff
@@ -28,11 +27,12 @@ public static class EcoreAnimator
         static readonly Dictionary<int, Func<bool>> _taskMap = new Dictionary<int, Func<bool>>();
         static readonly Object _taskLock = new Object();
         static int _newTaskId = 0;
-
         static Interop.Ecore.EcoreTaskCallback _nativeHandler;
-
         static EcoreAnimator()
```

Comment:
Initialize all static fields in 'EcoreAnimator' when those fields are declared and remove the explicit static constructor